src/test/java/ai/rev/speechtotext/integration/CustomVocabulariesTest.java

```diff
@@ -27,7 +27,10 @@
   @Test
   public void
       submitCustomVocabulary_MetadataAndVocabAreIncluded_ReturnCustomVocabularyInformationInProgress() {
```

Comment:
```suggestion
      submitCustomVocabulary_VocabAndAllOptionsAreIncluded_ReturnCustomVocabularyInformationInProgress() {
```

Need to rename the test to indicate that all options are being specified, feel free to use an alternate phrasing to my suggestion.